src/SearchPanel.jsx

```diff
@@ -1,14 +1,31 @@
 import './SearchPanel.less'
+import { connect } from 'react-redux'
+import { FormattedDate } from 'react-intl'
 import PillPanel from './PillPanel'
 import React from 'react';
 import SearchBar from './SearchBar'
+import { shortIsoFormat } from './utils'
 
-export const SearchPanel = () =>
-    <div className="search-panel">
-      <h2>Search complaint data</h2>
-      <SearchBar />
-      <PillPanel />
-    </div>
+export class SearchPanel extends React.Component {
+  render() {
+    var lastUpdatedMessage = null;
 
+    if ( this.props.lastUpdated ) {
+      lastUpdatedMessage = <span className="date-subscript">(last updated: <FormattedDate value={ this.props.lastUpdated } />)</span>
```

Comment:
This line might be dinged by the linter as too long

Reply:
I've fixed this issue in the latest commit, thanks for pointing it out!